src/callstack.cr

```diff
@@ -1,12 +1,4 @@
-{% if flag?(:win32) %}
-  struct CallStack
-    def self.skip(*args)
-      # do nothing
-    end
-  end
-
-  {% skip_file() %}
-{% end %}
+{% skip_file() if flag?(:win32) %}
```

Comment:
we can probably use `skip_file` here

Reply:
Yeah, I was waiting for #5493 to make this change all at once.